sap/cf_logging/record/simple_log_record.py

```diff
@@ -80,7 +80,6 @@ def format(self):
             stacktrace = format_stacktrace(stacktrace)
             record['stacktrace'] = stacktrace.split('\n')
             record['msg'] += "\n"
```

Comment:
Why the new line ? I would remove it as well.

Reply:
It's because the record['msg'] already has a value setup before that comming from the constructor:
https://github.com/SAP/cf-python-logging-support/blob/master/sap/cf_logging/record/simple_log_record.py#L75
` 'msg': self.getMessage(),`
and for better readability the new line is left.

Reply:
new line removed